samples/sample-stack/image/Dockerfile-stack

```diff
@@ -1,3 +1,25 @@
 # Dockerfile for building the stack
 
-# Explain all the Appsody stack environment variables
\ No newline at end of file
+# Explain all the Appsody stack environment variables
```

Comment:
think you can remove this line with this added PR, otherwise LGTM

Reply:
Done!